From 04c977a4cc7335be42a84c9570450954235ac5db Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:37 +0800
Subject: [PATCH 60/60] Revert "drm/panfrost: Put mapping instead of shmem obj
 on panfrost_mmu_map_fault_addr() error"

This reverts commit 2e760fe05d3ea5b5addb02691ec8a29e601dff56.
---
 drivers/gpu/drm/panfrost/panfrost_mmu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/panfrost/panfrost_mmu.c b/drivers/gpu/drm/panfrost/panfrost_mmu.c
index 13596961ae17f..7fc45b13a52c2 100644
--- a/drivers/gpu/drm/panfrost/panfrost_mmu.c
+++ b/drivers/gpu/drm/panfrost/panfrost_mmu.c
@@ -491,7 +491,7 @@ static int panfrost_mmu_map_fault_addr(struct panfrost_device *pfdev, int as,
 err_pages:
 	drm_gem_shmem_put_pages(&bo->base);
 err_bo:
-	panfrost_gem_mapping_put(bomapping);
+	drm_gem_object_put(&bo->base.base);
 	return ret;
 }
 
-- 
2.25.1

